projects/packages/videopress/src/class-data.php

```diff
@@ -237,15 +237,23 @@ public static function get_initial_state() {
 		// Tweak local videos data.
 		$local_videos = array_map(
 			function ( $video ) {
+				$video              = (array) $video;
 				$id                 = $video['id'];
 				$media_details      = $video['media_details'];
 				$jetpack_videopress = $video['jetpack_videopress'];
```

Comment:
I wonder if we should cast straight here:

```suggestion
				$media_details      = (array) $video['media_details'];
				$jetpack_videopress = (array) $video['jetpack_videopress'];
```

Reply:
I used the `if` to add the `$read_error`. If we cast straight here it is not possible to know if there is a problem.

Reply:
fair enough